Add leading and trailing quotation marks in DataDir and filename to allow spaces in DbDir

abagrov opened the pull request:
Fix for #29

mysticmind commented:
@abagrov Unit tests are failing on non-windows. Besides just adding the path in double quotes doesn't seem to be helping. Did you test with a similar use case as in the issue details?

abagrov committed:
Allow DbDir with spaces

abagrov commented:
@mysticmind Yeah, i said a little wrong, this PR fixes only path with spaces. As far as i understood, problem with special character is coming from initdb and we cant do much with this. But we can handle path with spaces - i added few corrections and tests to PR.
BTW, this PR should fix spaces problem on non-windows environments too, but new tests are failing with permission denied error on non-windows environments, which seems not related to fixes. I didnt manage to solve this, so i added only windows condition for these tests as a workaround. So lets assume this PR fixes windows environment at least 🙂
